db/databaseInitsql.sql

```diff
@@ -91,17 +91,21 @@ INSERT INTO Food (fid,rid,fname,price,daily_limit,category) VALUES (108,4,'Pork
 INSERT INTO Promotions(pid) VALUES (1);
 INSERT INTO Promotions(pid) VALUES (2);
 INSERT INTO Promotions(pid) VALUES (3);
-INSERT INTO FDS_Promotions(pid, promo_rate, promo_type, promo_cat, start_datetime, end_datetime, promo_min_cost, promo_max_discount_limit, promo_max_num_redemption, promo_details_text) VALUES (1, 1, 'PERCENT', 'DELIVERY', '01/01/2020 12:00:00', '30/12/2020 23:59:59', 10, null, null, 'Free delivery for all of 2020!');
+INSERT INTO FDS_Promotions(pid, promo_rate, promo_type, promo_cat, start_datetime, end_datetime, promo_min_cost, promo_max_discount_limit, promo_max_num_redemption, promo_details_text) 
+VALUES (1, 1, 'PERCENT', 'DELIVERY', '01/01/2020 12:00:00', '30/12/2020 23:59:59', 10, null, null, 'Free delivery for all of 2020!');
 INSERT INTO Restaurant_Promotions(pid, promo_rate, promo_type, promo_cat, start_datetime, end_datetime, promo_min_cost, promo_max_discount_limit, promo_max_num_redemption, promo_details_text, rid) VALUES (2, 10, 'DOLLAR', 'CART', '01/03/2020 12:00:00', '30/03/2020 12:00:00', 15, 15, 100, '10$ off with minimum order of 15$ at Western Food Store during March!', 4);
-INSERT INTO Restaurant_Promotions(pid, promo_rate, promo_type, promo_cat, start_datetime, end_datetime, promo_min_cost, promo_max_discount_limit, promo_max_num_redemption, promo_details_text, rid) VALUES (3, 5, 'DOLLAR', 'CART', '01/04/2020 12:00:00', '30/04/2020 12:00:00', 10, 5, 100, '5$ off with minimum order of 10$ at South XLB store during April! First 50 customers only!', 3);
+INSERT INTO Restaurant_Promotions(pid, promo_rate, promo_type, promo_cat, start_datetime, end_datetime, promo_min_cost, promo_max_discount_limit, promo_max_num_redemption, promo_details_text, rid) VALUES (3, 5, 'DOLLAR', 'CART', '01/04/2018 12:00:00', '30/06/2018 12:00:00', 10, 5, 100, '5$ off with minimum order of 10$ at South XLB store during April! First 50 customers only!', 3);
```

Comment:
can change back to 2020, trying to focus on data in 2020 better for demo

Reply:
done